tpm2/Makefile

```diff
@@ -1,13 +1,10 @@
 # Makefile for KMYTH using TPM 2.0
 
-# Choose OpenSSL 'base' path (header file and library path prefix)
-OPENSSL_DIR = $(MY_OPENSSL_DIR)#         custom shell environment variable
-
 # Specify the 'libkmythlogger' library path
-KMYTH_LOG_DIR = $(MY_KMYTH_LOG_DIR)#       custom shell environment variable
+LOGGER_DIR ?= logger#       		       Set a default path to the logger
```

Comment:
This path to the logger should be absolute. With this relative path, I believe that if the applications are invoked from any directory other than <path>/kmyth/tpm2, the logger library will not be found. One way to potentially fix this is by using "LOGGER_DIR ?= $(abspath logger)" instead.

Reply:
Would it be better to bundle the logger into the shared library files rather than trying to deal with making sure it can find `libkmythlogger` at runtime?